lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
                     event.preventDefault();
-                    alert("You clicked the button!");
-                    //placeholder; will route to delete team page
+                    var self = this;
+                    var msg = new PromptView.Warning({
+                        title: gettext('Delete this team?'),
+                        message: gettext('Deleting a team removes it from the team listing view, and removes members from the team as well.'),
+                        actions: {
+                            primary: {
+                                text: gettext('Delete'),
+                                class: 'action-delete',
+                                click: function () {
+                                    msg.hide();
+                                    self.handleDelete();
+                                }
+                            },
+                            secondary: {
+                                text: gettext('Close'),
+                                class: 'action-cancel',
+                                click: function () {
+                                    msg.hide();
+                                }
+                            }
+                        }
+                    });
+                    msg.show()
                 },
 
                 editMembership: function (event) {
                     event.preventDefault();
                     alert("You clicked the button!");
                     //placeholder; will route to remove team member page
+                },
+
+                handleDelete: function () {
+                    var self = this;
+                    this.team.destroy().then(function () {
```

Comment:
I agree that this should have some sort of feedback if the deletion can't be completed. I'm not sure what the best way to display that feedback is, though -- we have the standard system notifications, but we've also got a teams-specific error pattern for things like invalid URL fragments. @frrrances, @marcotuts?